test/lib/code_corps/github/sync/sync_test.exs

```diff
@@ -24,8 +24,49 @@ defmodule CodeCorps.GitHub.SyncTest do
 
   alias Ecto.Changeset
 
-  describe "installation_repositories_event/1 added" do
+  @doc """
+  Some clauses defined seem difficult or impossible to reach so their tests were ommitted
+    - {:error, :validation_error_on_syncing_installation, Changeset.t()}
+    - {:error, :validation_error_on_marking_installation_processed, Changeset.t()}
+    - {:error, :unexpected_transaction_outcome, any}
+  However, if these clauses can be caused by some updates upstream we should cover them with tests
+  """
```

Comment:
`@doc` is used generally for public function documentation only (`def`). This should just be a simple comment:

```Elixir
# Some clauses defined seem difficult or impossible to reach so their tests were omitted
#  - {:error, :validation_error_on_syncing_installation, Changeset.t()}
#  - {:error, :validation_error_on_marking_installation_processed, Changeset.t()}
#  - {:error, :unexpected_transaction_outcome, any}
# However, if these clauses can be caused by some updates upstream we should cover them with tests
```

Also, "omitted" is spelled with one `m`

Reply:
fixed this!